testing/ctorrent-dnh/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Denis Ryabyy <vv1r0x@gmail.com>
+# Maintainer: Denis Ryabyy <vv1r0x@gmail.com>
+pkgname=ctorrent-dnh
+pkgver=3.3.2
+pkgrel=0
+pkgdesc="Enhanced CTorrent is a BitTorrent client for unix-like environments"
+url="http://www.rahul.net/dholmes/ctorrent/"
+arch="all"
+license="GPL2"
+
+source="${pkgname}${pkgver}.tar.gz::https://sourceforge.net/projects/dtorrent/files/dtorrent/${pkgver}/${pkgname}${pkgver}.tar.gz/download
+unistd.patch"
+builddir="$srcdir"/${pkgname}${pkgver}
+
+prepare() {
+	default_prepare
+	./configure --prefix $pkgdir/usr
+}
+
+build() {
+	cd "$builddir"
+	make
+	make install
```

Comment:
Somehow when it in package() function binary not including into the package. But it in pkg dir if I run steps manually with abuild. I'll give you more detailed answer when be near my pc.